l10n_cn_hr_payroll/models/l10n_cn_hr_payroll.py

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8 -*-
+# © 2016 Elico corp (www.elico-corp.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp import fields, models, api, _
+import openerp.addons.decimal_precision as dp
+from openerp.exceptions import ValidationError
+
+
+class HrContractCn(models.Model):
+    _inherit = 'hr.contract'
+
+    social_insurance_amount = fields.Float(
+        'Social Insurance Base',
+        digits_compute=dp.get_precision('Payroll')
+    )
+    pit_base_amount = fields.Float(
+        'PIT Base',
+        digits_compute=dp.get_precision('Payroll'),
+        default=0
+    )
+    pit_exemption_amount = fields.Float(
```

Comment:
This field is useful. 
<record id="hr_rule_pit1" model="hr.salary.rule">
            <field name="category_id" ref="hr_payroll.DED"/>
            <field name="parent_rule_id" ref="hr_rule_pit"/>
            <field name="name">Personal Income Tax</field>
            <field name="code">PIT1</field>
            <field name="sequence">610</field>
            <field name="appears_on_payslip" eval="True"/>
            <field name="condition_select">range</field>
            <field name="condition_range">min(BFPIT,contract.pit_base_amount) - contract.pit_exemption_amount</field>
            <field name="condition_range_min">0</field>
            <field name="condition_range_max">1499.99</field>
            <field name="amount_select">code</field>
            <field name="amount_python_compute">result = ((min(BFPIT,contract.pit_base_amount) - contract.pit_exemption_amount) * 0.03)</field>
        </record>